src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -0,0 +1,92 @@
+# Managing Guice distributed James
+
+This guides aims to be an entry-point to the James documentation for user managing a distributed Guice James server.
+
+It includes:
+
+ - Simple architecture explanations
+ - Propose some diagnostic for some common issues
+ - Present procedures that can be set up to address these issues
+
+In order to not duplicate information, existing documentation will be linked.
+
+Please note that this product is under active development, should be considered experimental and thus is addressed to 
+advanced users.
+
+## Table of content
+
+ - [Overall architecture](#overall-architecture)
+ - [Basic Monitoring](#basic-monitoring)
+
+## Overall architecture
+
+Guice distributed James server intends to provide a horizontally scalable email server.
+
+In order to achieve this goal, this product leverages the following technologies:
+
+ - **Cassandra** for meta-data storage
+ - **ObjectStorage** (S3) for binary content storage
+ - **ElasticSearch** for distributed search
+ - **RabbitMQ** for messaging
+
+A [docker-compose](https://github.com/linagora/james-project/blob/master/dockerfiles/run/docker-compose.yml) file is 
+available to allow you to quickly deploy locally this product.
+
+## Basic Monitoring
+
+A given number of tools are available "out of the box" to help an administrator diagnose issues:
+ - [Structured logging into Kibana](#structured-logging-into-kibana)
+ - [Metrics graphs into Grafana](#metrics-graphs-into-grafana)
+ - [WebAdmin HealthChecks](#webadmin-healthchecks)
+
+### Structured logging into Kibana
+
+Read this page regarding [setting up structured logging](monitor-logging.html#Guice_products_and_logging).
+
+We recommend closely monitoring **ERROR** and **WARNING** logs. Those logs should be considered not normal.
+
+If you encounter some suspicious logs:
+ - If you have any doubt about the log being caused by a bug in James source code, please reach us via 
+ [the bug tracker](https://issues.apache.org/jira/browse/JAMES) or [the user mailing list](/mail.html)
+ - They can be due to insufficient performance from tier applications (eg Cassandra timeouts). In such case we advise
+ you a close review of performances at the database level.
```

Comment:
Any #issue_number for this?

Reply:
What do you mean?

Reply:
@naviat He is assuming there's maybe an error specific to the external services, then you should rather look at the external services instead of James

Reply:
That is what I mean by ` a close review of performances at the tier level` ;-)